interfaces/builtin/mpris.go

```diff
@@ -189,18 +187,23 @@ func (iface *MprisInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *i
 func (iface *MprisInterface) PermanentSlotSnippet(slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
 	switch securitySystem {
 	case interfaces.SecurityAppArmor:
-		snippet := mprisPermanentSlotAppArmor
+		name, err := iface.GetName(slot.Attrs)
```

Comment:
I started with private by I wanted to test GetName() specifically in mpris_test.go and wasn't sure how to do that without making it public. Is there another way? If not and you feel strongly about this, I suppose I could instead of testing GetName() specifically I could instead test getName() via PermanentSlotSnippet tests.